cluster-up/cluster/external/provider.sh

```diff
@@ -20,6 +20,7 @@ docker_tag=\${DOCKER_TAG}
 docker_prefix=\${DOCKER_PREFIX}
 manifest_docker_prefix=\${DOCKER_PREFIX}
 image_pull_policy=\${IMAGE_PULL_POLICY:-Always}
+kubectl=\${KUBECTL}
```

Comment:
Any reasons for adding this?

Reply:
For external cluster provider The --kubectl-path here https://github.com/kubevirt/kubevirt/blob/main/hack/functests.sh#L69 is getting set as empty. The reason for this behaviour is unlike k8s providers where the kubectl binary will be scp' ed from control node https://github.com/kubevirt/kubevirt/blob/main/cluster-up/cluster/k8s-provider-common.sh#L244, the external cluster don't have this capability. So, we have to manually export the env with KUBECTL variable. So, In order to set this env variable while running test, we have to source the KUBECTL env to provider config.

I will remove this change as the kubevirt downloads the latest kubevirtci tag and put those files in the cluster-up folder.  I have raised a pr in the kubevirtCI 
https://github.com/kubevirt/kubevirtci/pull/1238